extensions/open-in-json-hero/package.json

```diff
@@ -23,6 +24,14 @@
     }
   ],
   "preferences": [
+    {
+      "name": "tab",
+      "description": "Currently you can leave this blank or choose /editor or /tree",
+      "type": "textfield",
+      "required": false,
+      "default": "/editor",
```

Comment:
Maybe, if the options are limited and pre-defined, it should be a dropdown option, not a textfield, right?

Reply:
Yeh I had that originally but changed it to this as to be more flexible. If the site changes.

Reply:
there's is also no validation to see if the path is valid. Either you add the dropdown or add validation :)